integration_test/playertesting/PlayerTestWorld.ts

```diff
@@ -276,4 +276,13 @@ export default class PlayerTestWorld {
       delete this.eventListeners[key];
     };
   };
+
+  private ensurePlayerInitialized = async (): Promise<void> => {
+    if (this.isPlayerInitialized === false) {
+      // Trick to make sure the player is initialized,
+      // otherwise method calls might have no effect.
+      await new Promise((resolve) => setTimeout(resolve, 150));
```

Comment:
How come the change to the fixed wait time? Why not use the same trick as before with `player.isPlaying()`?

Unrelated to this PR but it would be very nice if `player.initialize()` function was blocking, then we could remove all of this.

Reply:
This was trial and error, `player.isPlaying()` didn't seem to work reliably on Android emulator.

Agree, `player.initialize()` being async would possibly solve this problem.

Reply:
I see, that's unfortunate. Out of interest what happened? Did it await for ever?

Reply:
Actually the opposite, it was not waiting long enough.

I will double-check it with using the latest changes from `development`

Reply:
Unfortunately, it didn't help. Initial events are missed somehow.
I will keep the current solution for now!